overlord/ifacestate/ifacemgr_test.go

```diff
@@ -196,7 +196,9 @@ func (s *interfaceManagerSuite) mockIface(c *C, iface interfaces.Interface) {
 }
 
 func (s *interfaceManagerSuite) mockSnap(c *C, yamlText string) *snap.Info {
-	sideInfo := &snap.SideInfo{}
+	sideInfo := &snap.SideInfo{
+		Revision: snap.R(-1),
```

Comment:
shouldn't make a difference but I prefer using the normal case if it works, so snap.R(1) ?